pydis_site/apps/api/serializers.py

```diff
@@ -151,12 +151,10 @@ def create(self, validated_data: dict) -> MessageDeletionContext:
         """
         messages = validated_data.pop('deletedmessage_set')
         deletion_context = MessageDeletionContext.objects.create(**validated_data)
-        for message in messages:
-            DeletedMessage.objects.create(
-                deletion_context=deletion_context,
-                **message
-            )
-
+        deleted_messages = [DeletedMessage(deletion_context=deletion_context, **message) for message in messages]
+        DeletedMessage.objects.bulk_create(
+            deleted_messages
+        )
```

Comment:
Wow, who wrote the original, super inefficient code? Crazy.

I think it would be nice if: 1. `deleted_messages` was a generator (even though Django casts it to a list anyhow, if they ever implement some form of batching, we will benefit from it) and 2. this would be a single line.

Reply:
Alright, fixed in the force push 2128dcf63088f10d8613a345ee4c028ec06df2c7